inscriptions/ISic030011.xml

```diff
@@ -109,9 +113,10 @@
                                 <placeName type="modern">Selinunte</placeName>
```

Comment:
as previous, add modern and ancient place (Selinunte/Selinus) with generic coordinates